samples/hello/src/com/microsoft/aad/adal/hello/Constants.java

```diff
@@ -32,16 +32,16 @@
     
     // AAD PARAMETERS
     // https://login.windows.net/tenantInfo
-    static final String AUTHORITY_URL = "https://login.windows.net/common";
+    static final String AUTHORITY_URL = "https://login.windows-ppe.net/common";
```

Comment:
PPE is the only setup for interactive flow. We can update this when it is available on prod.